File: Backend/src/main/kotlin/com/game/cubox/objects/Enums.kt
package com.game.cubox.objects

enum class ServerMessageType {
    WELCOME,
    JOINED,
    UPDATE,
    PLAYER_JOIN,
    PLAYER_EXIT,
    EXIT
}

enum class RoomState{
    NONE,
    OPEN,
    PLAYING,
    CLOSED
}

enum class EnemyState{
    NONE,
    CHASING,
    ATTACK,
    DEAD
}

enum class PlayerState{
    NONE,
    MOVING,
    SHOOTING
}

File: Game/Assets/Scripts/Manager/Networking/enums.cs
public enum InputDir
{
    NONE,
    FRONT,
    BACK,
    RIGHT,
    LEFT,
    FRIGHT,
    FLEFT,
    BRIGHT,
    BLEFT
}

public enum InputType
{
    NONE, MOVE, SHOOT
}

public enum ServerMessageType
{
    WELCOME,
    UPDATE,
    PLAYER_JOIN,
    PLAYER_EXIT,
    EXIT
}

public enum EnemyState
{
    NONE,
    CHASING,
    ATTACK,
    DEAD
}